distribution/systemd/hardened-vector.service

```diff
@@ -5,20 +5,24 @@
 # capabilities
 
 [Unit]
-Description=Vector
-Documentation=https://vector.dev
+Description="Vector - An observability pipelines tool"
+Documentation=https://vector.dev/
+Wants=network-online.target
 After=network-online.target
-Requires=network-online.target
 
 [Service]
-EnvironmentFile=-/etc/default/vector
+Type=exec
 User=vector
 Group=vector
+# Use `systemctl edit vector` override environment variables,
+# or place `.conf` files under `/etc/systemd/system/vector.service.d/`.
+Environment="LOG_FORMAT=json"
+Environment="VECTOR_CONFIG_YAML=/etc/vector/aggregator/vector.yaml"
```

Comment:
Did you mean to add these to the non-hardened service file too? Or, rather, I don't understand why they are only in this one.

Reply:
That's a whoopsie - copy pasted before I made changes to the non-hardened one. Fixed in 4d4e08553